tools/go-agent/tools/enhancement.go

```diff
@@ -88,7 +88,6 @@ func EnhanceParameterNames(fields *dst.FieldList, fieldType FieldListType) []*Pa
 
 			for _, n := range f.Names {
 				result = append(result, newParameterInfo(n.Name, f.Type))
-				break
```

Comment:
Could you help to add more test case on this?

Reply:
done~